### ISSUE TITLE
Doesn't load v1.5 pruned(7gb)

### ISSUE THREAD

**Issue by woover** [TheLastBen/fast-stable-diffusion#200]
It gives a C^ error instead of loading it.
Doesn't work with a link to gdrive to download it in the fast dreambooth as well.

**Comment by TheLastBen** [comment]
Because of the RAM limit, try --medvram

**Comment by woover** [comment]
> Because of the RAM limit, try --lowram

I tried replacing webui-user with -- lowvram, still getting C^ Error
are there any alternative ways to use --lowvram?

**Comment by TheLastBen** [comment]
--lowram not --low**v**ram

**Comment by mhnoni** [comment]
> --lowram not --low**v**ram

Where should I add that command? or how can I pass command to colab version of auto?
I added --lowram on the last 3 lines of colab Start stable-diffusion secion but didn't work

```
  if os.path.isfile(model):
    !python /content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py $share --disable-safe-unpickle --ckpt "$model"
  else:
    !python /content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py $share --disable-safe-unpickle --ckpt-dir "$model"
except:
   !python /content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py $share --disable-safe-unpickle
   
   
```

**Comment by TheLastBen** [comment]
try with the "relaxed" colab and see if it loads

**Comment by EnricoPf** [comment]
> try with the "relaxed" colab and see if it loads

I had the same problem and it did work successfully on the relaxed collab, is there any reason for that?
I really miss some features on the relaxed collab though, is it possible to upgrade it with negative prompts and such?

**Comment by TheLastBen** [comment]
"relaxed" colab is based on hlky that uses the optimizedSD mode, A1111 with --lowram should do the same

**Comment by EnricoPf** [comment]
> "relaxed" colab is based on hlky that uses the optimizedSD mode, A1111 with `--lowram` should do the same

I know it must be my ignorance, but is it possible to use this argument on colab? where do i put it?

**Comment by TheLastBen** [comment]
@EnricoPf in the last cell (click twice on it), add the argument at the end like this : (don't copy the lines, just add the argument

```
 if os.path.isfile(model):
    !python /content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py $share --disable-safe-unpickle  --ckpt "$model" --lowram
  else:
    !python /content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py $share --disable-safe-unpickle  --ckpt-dir "$model" --lowram
except:
   !python /content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py $share --disable-safe-unpickle --lowram
```

**Comment by EnricoPf** [comment]
I did try that, doesn't work still, it just consumes all the available RAM then crashes :(

**Comment by TheLastBen** [comment]
Yep, the free colab limitation, but you still can use the 4GB version, no big difference

**Comment by mhnoni** [comment]
> webui.py

it would be nice to find out how to do it for auto111 too, maybe you can dig in webui.py and see if we can change a line there so it will launch with lowram as default? I'm not sure why the auto colab is not taking the command. there should be a way to pass command there.

**Comment by TheLastBen** [comment]
tried to implement the OptimisedSD mode in A1111, but getting "hijack" errors.

**Comment by TheLastBen** [comment]
Pass that argument and test it if it doesn't kill the VRAM usage, if it doesn't, I'll activate it by default in shared.py

**Comment by mhnoni** [comment]
> Pass that argument and test it if it doesn't kill the VRAM usage, if it doesn't, I'll activate it by default in shared.py

no need to force it as default since that would slow it for the rest of the world, and low ram has its own disadvantage when it comes to visuals, this was only a test, since relax works fine, I say leave it as it is. 
Thanks btw for pointing out shared.py I will just change that if I need to use auto with 7GB ckpt.
